portfolio/pom.xml

```diff
@@ -24,7 +24,7 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-  		<groupId>com.google.code.gson</groupId>
+  	  <groupId>com.google.code.gson</groupId>
   	  <artifactId>gson</artifactId>
   	  <version>2.8.6</version>
 	  </dependency>
```

Comment:
Also this `</dependency>` line needs to be out-dented :)
Looks like you are also using 4 spaces instead of 2 (which is the style above).

Reply:
![image](https://user-images.githubusercontent.com/48307028/83462923-12442f80-a432-11ea-8df7-ae30c3d34ad4.png)
ah okay I don't  know why it looks so different in my editor -- I'll try to resolve this